components/gardener/extensions/deployment.yaml

```diff
@@ -33,6 +33,7 @@ kubectl:
 spec_template:
   <<: (( &temporary &template ))
   version: (( &temporary( .landscape.versions.gardener.extensions[n] ) ))
+  valuesOverwrite: (( &temporary( valid(.landscape.gardener.extensions[n].valueOverwrites) ? .landscape.gardener.extensions[n].valueOverwrites :nil ) ))
```

Comment:
The 'nil' value for the spiff templating language is `~`.

Reply:
I changed `nil` to `~`